vic/drivers/classic/src/alloc_veg_hist.c

```diff
@@ -38,29 +38,27 @@ alloc_veg_hist(int                nrecs,
 {
     int i, j;
 
-    (*veg_hist) = (veg_hist_struct **) calloc(nrecs, sizeof(veg_hist_struct *));
+    (*veg_hist) = calloc(nrecs, sizeof(*(*veg_hist)));
```

Comment:
@bartnijssen - can you advise on a better way to write this line?

Does this mean the same thing: 
```C
(*veg_hist) = calloc(nrecs, sizeof(**veg_hist));
```